sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -410,6 +429,21 @@ object RapidsConf {
     .booleanConf
     .createWithDefault(false)
 
+  // METRICS
+
+  val METRICS_LEVEL = conf("spark.rapids.sql.metrics.level")
+      .doc("GPU plans can produce a lot more metrics than CPU plans do. In very large " +
+          "queries this can sometimes result in going over the max result size limit for the " +
+          "driver. Supported values include DEBUG: which will enable all metrics supported and " +
```

Comment:
Nit: Suggest removing the `:` from the values since it's not immediately clear whether it's part of the value.